pom.xml

```diff
@@ -32,7 +32,7 @@
     <dependency>
       <groupId>uk.gov.ons.ctp.product</groupId>
       <artifactId>surveysvc-api</artifactId>
-      <version>10.49.0</version>
+      <version>10.49.1-SNAPSHOT</version>
```

Comment:
I think we'd want to generally point at the latest release versions for when we merge back to master